### PR TITLE
feat(lyrics-plus): copy Genius page lyrics

### PR DESCRIPTION
**Pull request by Lseoksee**
Just like the Synced and Unsynced pages, we implemented lyric copying on the Genius pages.

https://github.com/spicetify/spicetify-cli/assets/91320047/1417f890-7a40-4349-9fe1-41d768bc2525

### CODE REVIEW

**Review by kyrie25 — APPROVED** [review]

**Comment by Lseoksee** [comment]
Additionally, we added a function to cache lyrics on the Genius page.
Cached lyrics can be found on the Unsynced page, not on the Genius page.

File: CustomApps/lyrics-plus/Pages.js [hunk 1]
```diff
@@ -561,6 +561,13 @@ const GeniusPage = react.memo(
 				ref: c => (container = c),
 				dangerouslySetInnerHTML: {
 					__html: lyrics
+				},
+				onContextMenu: event => {
+					event.preventDefault();
+					const copylyrics = lyrics.replace(/<br>/g, "\n").replace(/<[^>]*>/g, "");
```
Comment (kyrie25) [inline CustomApps/lyrics-plus/Pages.js:567]: Wouldn’t this copy the text annotation also?

Comment (Lseoksee) [reply CustomApps/lyrics-plus/Pages.js:567]: Yes it is

Comment (kyrie25) [reply CustomApps/lyrics-plus/Pages.js:567]: That isn’t exactly copying lyrics though, I’d suggest removing text annotations for caching to Unsynced also.

Comment (Lseoksee) [reply CustomApps/lyrics-plus/Pages.js:567]: Does text annotations mean something like "[verse]"?

Comment (kyrie25) [reply CustomApps/lyrics-plus/Pages.js:567]: Text annotations means the explanation that Genius has when you click on a highlighted section of the lyrics

Comment (Lseoksee) [reply CustomApps/lyrics-plus/Pages.js:567]: If that's the case, I coded not to copy it.

Comment (Lseoksee) [reply CustomApps/lyrics-plus/Pages.js:567]: It is deleted through the corresponding syntax applied to both files.
```javascript
replace(/<[^>]*>/g, "")
```

**Review by kyrie25 — APPROVED** [review]

**Review by rxri — APPROVED** [review]

**Pull request merged**